translations/de/get5.phrases.txt

```diff
@@ -2,123 +2,235 @@
+    {
+        "de"            "Hinweis: Du wurdest aufgrund deiner Spielaktivität als bereit markiert. Gebe {1} ein falls du nicht bereit bist."
+    }
+    "YouAreNotReady"
+    {
+        "de"            "Du wurdest als NICHT bereit markiert."
+    }
     "WaitingForEnemySwapInfoMessage"
     {
-        "de"            "{1} gewinnt die Seitenwahl. Wartend auf {2} oder {3}."
+        "de"            "{1} hat die Messerrunde gewonnen und darf somit die Seite zum starten wählen. Dies muss durch die Eingabe von {2} oder {3} geschehen."
     }
     "WaitingForGOTVBrodcastEndingInfoMessage"
     {
-        "de"            "Die Map wird gewechselt sobald der GOTV broadcast geendet hat."
+        "de"            "Die Map wird gewechselt sobald die GOTV Übertragung beendet ist."
+    }
+    "WaitingForGOTVVetoInfoMessage"
+    {
+        "de"            "Die Map wird gewechselt sobald die Map Vetos von GOTV übertragen wurden."
     }
     "NoMatchSetupInfoMessage"
     {
-        "de"			"Es ist kein Match aufgesetzt"
+        "de"			"Es ist kein Spiel konfiguriert"
     }
     "YouAreNotAPlayerInfoMessage"
     {
-        "de"            "Du bist kein Spieler von diesem Match"
+        "de"            "Du bist kein Spieler dieses Spiels"
     }
     "TeamIsFullInfoMessage"
     {
         "de"            "Dein Team ist voll"
     }
     "TeamForfeitInfoMessage"
     {
-        "de"            "{1} schaffte es nicht fertig zu werden und gab auf."
+        "de"            "{1} schaffte es nicht sich rechtzeitig als bereit zu markieren und gab auf."
     }
     "MinutesToForfeitMessage"
     {
-        "de"            "{1} hat {2} Minuten um fertig zu werden oder sie werden aufgeben."
+        "de"            "{1} hat {2} Minuten um sich als bereit zu markieren oder wird automatisch aufgeben."
     }
     "SecondsToForfeitInfoMessage"
     {
-        "de"            "{1} hat {2} Sekunden um fertig zu werden oder sie werden aufgeben."
+        "de"            "{1} hat {2} Sekunden um sich als bereit zu markieren oder wird automatisch aufgeben."
     }
     "10SecondsToForfeitInfoMessage"
     {
-        "de"            "{1} hat 10 Sekunden fertig zu werden oder sie geben auf."
+        "de"            "{1} hat 10 Sekunden um sich als bereit zu markieren oder wird automatisch aufgeben."
     }
     "MaxPausesUsedInfoMessage"
     {
-        "de"            "Dein Team hatte bereits die maximal zulässigen ({1}) Pausen."
+        "de"            "Dein Team hat bereits die maximal zulässigen Pausen ({1}) genutzt."
     }
     "MaxPausesTimeUsedInfoMessage"
     {
-        "de"            "Dein Team hat bereits die maximalen ({1}) der Pausezeit genutzt."
+        "de"            "Dein Team hat bereits die maximale Pausezeit ({1}) genutzt."
     }
     "MatchPausedByTeamMessage"
     {
-        "de"            "{1} hat das Match pausiert."
+        "de"            "{1} hat das Spiel pausiert."
+    }
+    "MatchTechPausedByTeamMessage"
+    {
```

Comment:
It shouldn't be "pause because of a technical problem", just "technical pause". Can we make that work in German?

Reply:
We could translate it into "Technische Pause" but that would be colloquial imho. But if we have the need to make it shorter, we could do that.

Reply:
I've asked on the dc for opinions on this one, lets wait if someone answers. The rest is fixed.

Reply:
done